cfme/tests/intelligence/reports/test_views.py

```diff
@@ -46,6 +42,8 @@ def test_report_view(report, view_mode):
         caseimportance: high
         initialEstimate: 1/6h
     """
-    view = navigate_to(report, 'Details')
+    view = navigate_to(report, "Details")
```

Comment:
Here is where you can pass `force=True` to force navigation instead of a browser refresh.  Since this is the first step in the test, and you added a refresh resetter, I'm assuming the fixture is leaving the browser on this page already, and you want to refresh it.

If you really do want/need a browser refresh in this instance, you can make a browser refresh call here, instead of impacting all navigation to Report details with a resetter method that refreshes.

The `force` option should update the view, through navigation, refreshing the content in the process to account for the fixture behavior.

Reply:
@mshriver We've already done all that you mentioned here, refer this [review](https://github.com/ManageIQ/integration_tests/pull/8519#discussion_r261662321) by Angelina. I guess refreshing the browser for this particular test might be the way to go.